ui/oopsyraidsy/oopsyraidsy_config.js

```diff
@@ -21,6 +22,7 @@ UserConfig.registerOptions('oopsyraidsy', {
         en: 'Number of mistakes to show in combat',
         de: 'Anzahl der Fehler, die während des Kampfes angezeigt werden',
         fr: 'Nombre de fautes à afficher en combat',
+        ja: '戦闘の時に表示する最大の間違い回数',
```

Comment:
This should be "戦闘中ミスをした回数".

Reply:
Do you think ミス is better than 間違い to describe mistakes in this situation?

I have written that before but one of my Japanese friends told me that ミス was ambiguity sometimes. What do you think about that?

Reply:
I think this option refers to the display-able maximum number of mistakes that players did in combat. If the number of their mistakes was more than this value, then the over part would not be displayed. So it could not be simply 戦闘中ミスをした回数.

it is more like 戦闘中に記録したミス回数がその数値を超えた場合は表示できません。

Reply:
I think those two words mean the same thing. If you want to use 間違い," 戦闘中に表示する間違えた回数". Most of Japanese use ミス here imo. so I suggested.

Reply:
Oh sorry, I misunderstand your suggested translation. Yours is right.
And if most Japanese use ミス, I would use that, too.
Thank you very much for your help again.

Reply:
Nah maybe you saw my comment before I edited. My bad!